workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,17 +49,86 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
+        {
+            var command = Common.SpatialBinary;
+            var commandArgs = "local worker launch UnityClient default";
+
+            if (Application.platform == RuntimePlatform.OSXEditor)
+            {
+                command = "osascript";
+                commandArgs = $@"-e 'tell application ""Terminal""
+                                     activate
+                                     do script ""cd {SpatialProjectRootDir} && {Common.SpatialBinary} {command}""
+                                     end tell'";
+            }
+
+            var processInfo = new ProcessStartInfo(command, commandArgs)
+            {
+                CreateNoWindow = false,
+                UseShellExecute = true,
+                WorkingDirectory = SpatialProjectRootDir
+            };
+
+            var process = Process.Start(processInfo);
+
+            if (process == null)
+            {
+                Debug.LogError("Failed to start a standalone client locally.");
+                return;
+            }
+
+            process.EnableRaisingEvents = true;
+            process.Exited += (sender, args) =>
+            {
+                // N.B. This callback is run on a different thread.
+                if (process.ExitCode == 0)
+                {
+                    return;
+                }
+
+                var logPath = Path.Combine(SpatialProjectRootDir, "logs");
+                var latestLogFile = Directory.GetFiles(logPath, "external-default-unityclient.log")
```

Comment:
is it possible to extract the logpath directly from the `spatialos.UnityClient.worker.json` file?